NarutoLutaDeNinjas-Map/src/main/java/br/tec/db/NinjaDeTaijutsu.java

```diff
@@ -8,29 +8,24 @@ public NinjaDeTaijutsu(String nome, int vida) {
 
     @Override
     public void usarJutsu(String jutsuKeyMap, Personagem personagemAtacado) {
-        if (this.getJutsus().containsKey(jutsuKeyMap)) {
+        if (this.verificaPelaKeySeOJutsoExisteESeTemChakra(jutsuKeyMap)
+                && personagemAtacado.verificaSeTemVida(personagemAtacado)) {
 
-            Jutsu jutsu = getJutsus().get(jutsuKeyMap);
-
-            if(verificaSeTemChakra(jutsu)) {
-                System.out.println(getNome() + " atacando usando Taijutsu\n");
-                perdeChakra(jutsu.getConsumoDeChakra());
-
-                if (personagemAtacado.validaAVidaAposDano(jutsu)) {
-                    if (desviar()) {
-                        jutsu.atacaOutroNinja(personagemAtacado);
-                    } else {
-                        System.out.println(personagemAtacado.getNome() + " desviou\n");
-                    }
-                }
+            System.out.println(this.getNome() + " atacando o " + personagemAtacado.getNome());
+            if (!desviar()) {
+                this.atacaOutroNinja(jutsuKeyMap, personagemAtacado);
             }else{
-                System.out.println(getNome() + " sem chakra");
+                this.perdeChakra(this.getJutsus().get(jutsuKeyMap).getConsumoDeChakra());
```

Comment:
Aqui você usou o perdeChakra da classe pai, seria super e não this. 
super: implementação do pai
this: implementação da classe